Hopsan-release-notes.txt

```diff
@@ -26,15 +31,77 @@
+- All Hopsan source files now use the file suffix '.cpp' rather then mixing '.cpp' and '.cc'.
+- Update Qt to 5.6.3 on Windows (official Windows 10 support).
+- Update the 64-bit Windows compiler to MinGW 4.9.4 (from 4.9.3). If you have manually set up a build environment for
+  custom component libraries you are encouraged to update (use one of the Hopsan downloads with compiler included).
+- All binaries now use lower-case names (hopsancli.exe instead of HopsanCLI.exe).
+- Continuous integration (Travis CI) now builds builds and tests every change to the Hopsan code. This will reduce the
+  risk of new code breaking old functionality.
+
+--- Enhancements ---
+
+- Optimization:
+  - Improved script file handling.
+  - Improved variable names in plots.
+  - Points are now sorted by objective value with DE algorithm.
+  - Script generation is now shown in a separate wizard.
+  - Reduced terminal output during optimization.
+  - New algorithm, Genetic Algorithm.
+
+- FMU:
+  - Exported Hopsan FMUs will now report Warning and Error messages. Info messages will be reported as Debug type.
+  - Imported FMUs will now have their messages redirected to the Hopsan core logger instead of standard output.
+    messages should now be visible inside HopsanGUI. Enable Debug messages to see FMU Info messages.
+
+- HopsanGUI:
+  - Add PI constant 'pi()' in HCOM script.
+  - Export plot to image dialog will close after successful export.
+  - Generator import/recompilation dialogs will now auto close if the import succeeds. Various improvement have also
+    been made to the generator error handling to prevent importing old leftover files if compilation fails.
+  - Improved Hopsan core log file handling, the log file is now put in the temporary directory next to log cache files.
+  - Debug messages with log data collection speed has been added to help debug severe slowdown problems after the
+    simulation is finish. Enable debug messages to show them.
+
+- HopsanCLI:
+  - The HopsanCLI takes a new argument '--log.corelogfile' to specify core log destination.
+    If not specified, the current working directory will be used.
+  - It is now possible to choose what ports to log when simulating using HopsanCLI.
+    Use the '--logonly' command line argument, and specify a comma-separated list of full port names.
+    It is also possible to alternatively specify a text file with one full port name per line.
+    Examples: Gain#out or SubsystemName#Gain#out
+  - When saving results from a HopsanCLI simulation, interface ports (system ports) will also be saved.
+  - Results saved by HopsanCLI now use scientific notation, this may increase precision for small values.
+
+--- Bug Fixes ---
+
+- Optimization:
+  - Plot windows should be reusable.
+  - Progress bars do not show with DE algorithm.
+  - Excessive progress bars are not removed when reducing number of models.
+  - Total progress bar is not updated with PSO algorithm.
+  - Abort button does only work once.
```

Comment:
Actually this still happens, but less often than before. I think we can remove or reformulate this line.

Reply:
If it does not work then that issue should be reopened, i will remove "abort buton only work once" from the releas notes